assets/config.php

```diff
@@ -187,6 +188,7 @@
      */
     'cache' => [
         'tag_base' => 'tenant', // This tag_base, followed by the tenant_id, will form a tag that will be applied on each cache call.
+        'prefix_base' => 'tenant_' // this prefix_base, followed by the tenant_id, will form a cache prefix that will be applied on each cache call
```

Comment:
```suggestion
        'prefix_base' => 'tenant_' // This prefix_base, followed by the tenant_id, will form a cache prefix that will be applied on each cache call.
```

Consistency with the above line.

Also, maybe `that will be used for every cache key` might make more sense? I used "applied on each *call*" for the `tag_base` because tags are applied on calls. But prefixes are used for keys.

Reply:
Makes sense. I updated.